docusaurus/docs/dev-docs/configurations/future-flags.md

```diff
@@ -0,0 +1,106 @@
+:::
+
+<!-- ! Commented out as not relevant for now -->
+<!-- Future flags can also be utilized for enabling coming breaking changes in upcoming versions (when prefixed by `vX`, with 'X' being the target version). In this scenario, if you decide to enable a future flag for a breaking change, you will need to migrate your application to adapt to this breaking change. -->
+
+## Enabling a future flag
+
+To enable a future flag:
+
+0. (_optional_) If the server is running, stop it with `Ctrl-C`.
+1. Open the `config/features.js|ts` file or create it if the file does not exist yet. The file will export a `future` object with all the future flags to enable.
+2. To enable a future flag, add its property name (see [full list](#available-future-flags)) to the `future` object and ensure the property's value is set to `true`. The following example shows how to enable the `contentReleases` future flag:
+
+  <Tabs groupId='js-ts'>
+
+  <Tab value="js" label="JavaScript">
+
+  ```ts title="/config/features.ts"
+  module.export = ({ env }) => ({
+    future: {
+      // You could also simply write: contentReleases: true
+      contentReleases: env.bool('STRAPI_FEATURES_FUTURE_CONTENT_RELEASES', false),
+    },
+  })
+
+  ```
+
+  This example assumes that you have an `.env` environment file at the root of your application and that the file includes the following line:
+
+  ```json title=".env"
+  STRAPI_FEATURES_FUTURE_CONTENT_RELEASES=true
+  ```
+
+  If your environment file does not include this value, the `contentReleases` future flag property value  will default to `false` and the experimental feature will not be enabled.
+
+  </Tab>
+
+  <Tab value="ts" label="TypeScript">
+
+  ```ts title="/config/features.ts"
+  export default {
+    future: {
+      // You could also simply write: contentReleases: true
+      contentReleases: env.bool('STRAPI_FEATURES_FUTURE_CONTENT_RELEASES', false),
+    },
+  };
+  ```
+
+  This example assumes that you have an `.env` environment file at the root of your application and that the file includes the following line:
+
+  ```json title=".env"
+  STRAPI_FEATURES_FUTURE_CONTENT_RELEASES=true
+  ```
+
+  If your environment file does not include this value, the `contentReleases` future flag property value will default to `false` and the experimental feature will not be enabled.
+
+  </Tab>
+  </Tabs> 
+
+3. Rebuild the admin panel and restart the server:
+
+  <Tabs groupId="yarn-npm">
+  <Tab value="yarn" label="Yarn">
+  
+    ```sh
+      yarn build && yarn develop
+    ```
+  </Tab>
+  <Tab value="npm" label="NPM">
+
+    ```sh
+      npm run build && npm run develop
+    ```
+
+  </Tab>
+  </Tabs>
+
+## Future flags API
+
+Developers can use the following APIs to interact with future flags:
+
+- Features configuration is part of the `config` object and can be accessed with `strapi.config.get('features')`.
```

Comment:
This wil _read_ the features object and is the same thing as strapi.features.config

![Screenshot 2023-12-18 at 11 05 43](https://github.com/strapi/documentation/assets/26598053/ee012930-a5d0-427e-8efc-e22b9c999b17)

strapi.features.future returns a function to determine if it is enabled (this is what we use for the content-releases plugin)

![Screenshot 2023-12-18 at 11 05 16](https://github.com/strapi/documentation/assets/26598053/ce6bcd54-e25c-426c-8473-ec1aa20c053c)

![Screenshot 2023-12-18 at 11 07 26](https://github.com/strapi/documentation/assets/26598053/7d8ba35c-082c-471e-8aa0-d74bd3fe8205)